modules/ding_reservation/ding_reservation.module

```diff
@@ -128,17 +145,30 @@ function ding_reservation_ding_entity_buttons($type, $entity, $view_mode, $widge
                   'use-ajax',
                 ),
                 'id' => 'reservation-' . $entity->id,
+                'data-entity-id' => $entity->id,
```

Comment:
It seems that these were used by our workarounds and ding_availability.js to find the reservation buttons add by ding_reservation. Since this is removed now, I think we could safely remove it now. 

I'll look into it.